src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/Value.tsx

```diff
@@ -21,10 +21,11 @@ const InlineText = styled(Text)`
 const NestedWrapper = styled.div`
   text-indent: 24px;
 `
+type paramType = 'address' | 'tx'
 
 interface RenderValueProps {
   method: string
-  type: string
+  type: string | paramType
```

Comment:
do we have any type besides `address` or `tx`?

Reply:
Yes, it could be `bool` for example. But for etherscan link to work only `tx` or `address` are allowed.

Reply:
If only tx or address are allowed, why do we allow string?